modules/openy_features/openy_location/modules/openy_loc_filter/openy_loc_filter.info.yml

```diff
@@ -0,0 +1,11 @@
+name: Open Y Location filter
```

Comment:
Let's add a reference to the settings page, so we can see it on the Extend page in Drupal 
![image](https://user-images.githubusercontent.com/6285570/75802615-0ba01080-5d8e-11ea-81aa-9dae9af398ac.png)